docker-compose.yaml

```diff
@@ -31,13 +31,16 @@ services:
       dockerfile: splinterd/Dockerfile-installed-${DISTRO}
       args:
         - REPO_VERSION=${REPO_VERSION}
+        - CARGO_ARGS=${CARGO_ARGS}
```

Comment:
The commit message "Add example splinter db url to splinter-node-0 image." doesn't seem to exactly reflect what is in this commit.

Reply:
updated